README.md

```diff
@@ -5,4 +5,26 @@
 
 # Google Container Tools for IntelliJ
 
-In development
+This plugin integrates continuous development on [Kubernetes](https://www.kubernetes.io) using [Skaffold](https://skaffold.dev) into [Jetbrains family of IDEs](https://www.jetbrains.com/products.html), including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm and WebStorm.
+
+**Note**: This plugin is still in development.
+
+## Features
+
+* One click deployment to Kubernetes clusters right from your IDE using kubectl, Helm, Google Cloud Build, Jib and Kanico using Skaffold integration with IDE.
```

Comment:
suggestion (current wording makes it seem like you must use Helm, GCB etc.):
> One click deployment to Kubernetes clusters right from your IDE using Skaffold (link to Skaffold github). Configure Skaffold to use your desired build and deployment strategies: works with kubectl, Helm, Google Cloud Build (for remote builds), Jib and Kanico.

Reply:
done.